blog/2023-06-12-bringing-dlc-to-lightning-part-1.md

```diff
@@ -0,0 +1,89 @@
+---
+slug: dlc-to-lightning-part-1
+title: Bringing DLC to Lightning Part 1
+authors: [richard]
+tags: [DLC, lightning, knowledge-base]
+---
+
+At 10101 we are extending the Lightning network to support DLCs, allowing traders not only to pay and receive payments, but also to trade right from their channels.
```

Comment:
Maybe? It sounds a bit more ambitious.

```suggestion
At 10101 we are extending the Lightning network to support DLCs, allowing Bitcoiners not only to pay and receive payments, but also to trade right from their channels.
```

Also perhaps we could say that we are "working on extending", so that it doesn't sound like we are the only ones doing it. Or something like that.